doc-files/release-notes-17.0.2.md

```diff
@@ -0,0 +1,33 @@
+# Release Notes for JavaFX 17.0.2
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+As of JDK 11 the JavaFX modules are delivered separately from the JDK. These release notes cover the standalone JavaFX 17.0.2 release. As such, they complement the [JavaFX 17 Release Notes](https://github.com/openjdk/jfx/blob/jfx17/doc-files/release-notes-17.md).
+
+JavaFX 17.0.2 requires JDK 11 or later.
+
+List of Task
+------------
+
+| Issue Key                                                       | Summary                                                 | Subcomponent |
+| --------------------------------------------------------------- | ------------------------------------------------------- | ------------ |
+| [JDK-8272638](https://bugs.openjdk.java.net/browse/JDK-8272638) | Update copyright header for files modified in 2021      | other        |
+| [JDK-8274413](https://bugs.openjdk.java.net/browse/JDK-8274413) | FX: Update copyright year in docs, readme files to 2022 | other        |
+| [JDK-8279396](https://bugs.openjdk.java.net/browse/JDK-8279396) | Define version in .jcheck/conf                          | other        |

```

Comment:
Hi @kevinrushforth ,

Can you confirm if issues like this needs to be in the release note. The issue itself is labelled as `noreg-cleanup`, however, the backport issue for openjfx17.0.1 doesn't have any such label.